index.js

```diff
@@ -10,9 +10,39 @@ function parseGitUri(uri) {
     uri.match(/git@github.com:(.*)\/(.*)\.git/);
 }
 
+function verifyChangelog() {
+  var changelog = exec('git diff HEAD~1');
+  var lines = changelog.split('\n');
+  var isOneLine = (lines.length === 1);
+  var deletions = 0;
+  var containsErrorMessage = false;
+  lines.filter(function (line) {
+    if (/^-/.test(line)) deletions++;
+    if (line.indexOf('Make a POST first') >= 0) containsErrorMessage = true;
+  });
+  if (isOneLine) {
+    revertChanges();
+    console.error('Changelog diff should be more than 1 line long');
+    process.exit(3);
+  }
+  if (deletions > 10) {
```

Comment:
I checked the history to see how many deletions there are typically when the changelog is updated:
```
$ commits=`git log --oneline --before="6 days ago" | grep "docs(changelog)" | grep -Po '^\w+'`
$ for c in $commits; do echo $c; git diff $c~1..$c | grep '^-[^-]'; echo; done
2089d3e
-[Full Changelog](https://github.com/shelljs/shelljs/compare/v0.7.4...HEAD)

280c7fc

3986d61

e24b675
-## [Unreleased](https://github.com/shelljs/shelljs/tree/HEAD)
-[Full Changelog](https://github.com/shelljs/shelljs/compare/v0.7.2...HEAD)

cec859e

413bc79
-## [Unreleased](https://github.com/shelljs/shelljs/tree/HEAD)
-[Full Changelog](https://github.com/shelljs/shelljs/compare/v0.7.0...HEAD)

430042f

```
It looks like there should be no more that 2 deletions, so I think that 2 should be the limit.